src/components/Personality/PersonalityCard.jsx

```diff
@@ -99,6 +126,27 @@ class PersonalityCard extends Component {
                                         }
                                         href={`${this.props.hrefBase ||
                                             "personality/"}${personality._id}`}
+
+                                        style={{
+                                            background: "#2D77A3",
```

Comment:
In this case, you need to change the button's border color as well. It's using the default color.